packages/api/src/test-runner/run-result-helpers.ts

```diff
@@ -11,16 +11,17 @@ export function determineHitLimitReached(hitCount: number | undefined, hitLimit:
   return;
 }
 
-export function toMutantRunResult(dryRunResult: DryRunResult): MutantRunResult {
+export function toMutantRunResult(dryRunResult: DryRunResult, reportAllKillers = false): MutantRunResult {
```

Comment:
>I added the boolean because if `toMutantRunResult` starts returning an array it'd break some of the test from other test runners. For example this test
> 
> So I think it is necessary while we implement this for the other tests runners.
> 
> What do you think?

Ah I understand. Great thinking. I think it's OK for `@stryker-mutator/api@5.4` to not be compatible with `@stryker-mutator/core@5.3`. This won't be a problem at all for Stryker, because we're pinning the dependency on `@stryker-mutator/api` with each release. If a test runner in the wild is using `@stryker-mutator/api`, than it should also pin the version on minor version. 

However, you are correct that it is strictly a breaking change. Feel free to keep it like this.

Reply:
I'll keep it like this if you don't mind. When all is done in the rest of the test runners it'll be easy to remove.